pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
Jimmi after your rebase go ahead and squash this down to whatever the
minimal is.